apps/showcase/src/services/webcontainer/webcontainer.helpers.ts

```diff
@@ -4,6 +4,23 @@ import { Terminal } from '@xterm/xterm';
 import { MonacoTreeElement } from 'ngx-monaco-tree';
 import { BehaviorSubject } from 'rxjs';
 
+/**
+ * Check if the monaco tree contains the path in parameters
+ *
+ * @param tree
+ * @param path
+ */
+export function checkIfPathInMonacoTree(tree: MonacoTreeElement[], path: string[]): boolean {
+  if (path.length === 0) {
+    return false;
+  }
+  const treeWithPath = tree.find((treeElement) => treeElement.name === path[0]);
+  if (!treeWithPath) {
+    return false;
+  }
+  return path.length === 1 ? true : checkIfPathInMonacoTree(tree, path.slice(0, -1));
```

Comment:
shouldn't this be:
```suggestion
  return path.length === 1 ? true : checkIfPathInMonacoTree(treeWithPath, path.slice(1));
```
?